sdk/python/tests/test_client.py

```diff
@@ -1,26 +1,24 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v.2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
-
 import subprocess
 import sys
 import os
 import pytest
+import time
 
-import client.client as es
+import embeddings as es
 
 
 @pytest.fixture
 def embedding_store_proc():
-    proc = subprocess.Popen(os.environ["TEST_SRCDIR"] +
-                            "/__main__/embeddingstore/main")
+    # TODO: better dynamic binary path setting/finding
+    proc = subprocess.Popen(os.path.join(os.environ["EMBEDDINGS_DIR"], 'bazel-out/darwin-fastbuild/bin/embeddingstore/main'))
+    time.sleep(0.3)
```

Comment:
We can – I want to avoid creating unexpected behavior for users (e.g. blocking while waiting for retries), so I think we should come back to this once more of the client is fleshed out.